_posts/en/newsletters/2018-08-14-newsletter.md

```diff
@@ -35,7 +35,22 @@ projects.
 
 ## Dashboard items
 
-<!-- TODO: @moneyball -->
+{% assign img1_label = "Transactions per block, 25-block moving average, July 14, 2018 - August 13, 2018" %}
+
+- Transaction [fees remain very low][fee metrics]: Anyone who can wait 10 or
+more blocks for confirmation can reasonably pay the default minimum fee rate.
+It’s a good time to [consolidate UTXOs][consolidate info].
+
+- [BTC hash rate][btc hash rate] briefly touched 60 EH/s on August 10, and has a 7 day average of 48 EH/s.
+
+- One way to measure activity in Bitcoin is to look at the the number of transaction in each 
+block. This metric is vaguely periodic in that it has peaks at around 13:00 to 17:00 UTC each
+day. The graph below shows a 25-block moving average of the number of transactions. It was sourced from the
+[Optech beta dashboard][periodic txn data] which we encourage people to try out and provide us feedback about.
```

Comment:
Grammar nit: s/which/that/ unless you're writing British English, in which case it's all good.

Reply:
Will fix